packages/ckeditor5-table/src/plugin.ts

```diff
@@ -0,0 +1,139 @@
+/**
+ * @license Copyright (c) 2003-2023, CKSource Holding sp. z o.o. All rights reserved.
+ * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
+ */
+
+import type {
```

Comment:
This looks like a duplication of index.ts. Do we really need that?

Reply:
On today's daily, the team agreed to keep module augmentation in a separate `augmentation.ts` file.